controllers/hibp.js

```diff
@@ -4,35 +4,52 @@ const AppConstants = require("../app-constants");
 const DB = require("../db/DB");
 const EmailUtils = require("../email-utils");
 const HIBP = require("../hibp");
-
+const sha1 = require("../sha1-utils");
+const HBSHelpers = require("../hbs-helpers");
 
 async function notify (req, res) {
+
+  const hashes = req.body.hashSuffixes.map(suffix=>req.body.hashPrefix + suffix);
+  const subscribers = await DB.getSubscribersByHashes(hashes);
```

Comment:
Nit: these two lines may need to move back down. We had them later in the method after the `throw` to avoid a potentially unnecessary DB call.